src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -100,52 +100,61 @@ object DeploymentValidation extends SparkSessionWrapper {
 
   private def validateMountCount(conf: MultiWorkspaceConfig): DeploymentValidationReport = {
 
-    val testDetails =
-      s"""WorkSpaceMountTest
-         |APIURL:${conf.api_url}
-         |DBPATWorkspaceScope:${conf.secret_scope}
-         |SecretKey_DBPAT:${conf.secret_key_dbpat}""".stripMargin
-    try {
-      val patToken = dbutils.secrets.get(scope = conf.secret_scope, key = conf.secret_key_dbpat)
-      val apiEnv = ApiEnv(false, conf.api_url, patToken, getClass.getPackage.getImplementationVersion)
-      val endPoint = "dbfs/search-mounts"
-      val mountCount = ApiCallV2(apiEnv, endPoint).execute().asDF().count()
-      if(mountCount<50)
-        {
+    if (conf.cloud.toLowerCase == "azure" &&  conf.workspace_id.trim != spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId")) {
```

Comment:
Please add some docs around this if statement and even the func would be helpful.

I think it would also be helpful to convert the section below into a variable that is more english legible such as 
```scala 
val isAzure = conf.cloud.toLowerCase == "azure"
val isRemoteWorkspace = conf.workspace_id.trim != Initializer.getOrgId
if (isAzure && isRemoteWorkspace) ...
```
I think it makes the if much more legible.
```scala
conf.workspace_id.trim != spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId"
```

Reply:
shouldn't there be one more case added to the if statement as well? and mount_mapping_path is null or something like that to disable this validation if a mount_mapping_path was added?

Note: This should only be disabled if the mount_mapping_path is provided AND has org_id for the conf.workspace_id

Reply:
implemented for mount_mapping_path validation will implement in a different branch once this branch is merged

Reply:
for mount_mapping_path validation i will have it in a different branch once this PR is merged